src/app/api/workout/[workoutId]/favorite/route.ts

```
@@ -0,0 +1,55 @@
+    }
+    
+    // TypeScript workaround - cast to any to handle new field
+    const currentFavoriteStatus = (existingWorkout as any).favorite || false
+    
+    // Toggle the favorite status
+    const updatedWorkout = await prisma.workout.update({
+      where: { id: workoutId },
+      data: { 
+        // TypeScript workaround - use unknown key in object
+        favorite: !currentFavoriteStatus
+      } as any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)